packages/react/src/hooks/use-gdpr-consent.ts

```diff
@@ -0,0 +1,128 @@
+import { BaseUser, GDPRConsent, GDPRConsentLevel } from "@slashid/slashid";
+import { useCallback, useEffect, useMemo, useState } from "react";
+import { isBrowser } from "../browser/is-browser";
+import { useSlashID } from "./use-slash-id";
+
+export const STORAGE_GDPR_CONSENT_KEY = "@slashid/GDPR_CONSENT";
+
+interface GDPRConsentStorage {
+  getConsentLevels: () => Promise<GDPRConsent[]>;
+  setConsentLevels: (
+    consentLevels: GDPRConsentLevel[]
+  ) => Promise<GDPRConsent[]>;
+  deleteConsentLevels: () => Promise<void>;
+}
+
+export const createLocalGDPRConsentStorage = (): GDPRConsentStorage => ({
+  getConsentLevels: async () => {
+    if (!isBrowser()) {
```

Comment:
This can be the responsibility of the parent scope - we can check it there and only create an instance if we're in a browser, no need to have local checks then.